source/reference/methods.txt

```diff
@@ -827,12 +827,22 @@ Native Methods
 
      - Description
 
-   * - ``isInteractive()`` 
+   * - .. _mongosh-native-method-cd:
+
+       ``cd()``
+
+     - Changes the current working directory to the specified path.
+
+   * - .. _mongosh-native-method-isInteractive:
+
+       ``isInteractive()``
 
      - Returns a boolean indicating whether mongosh is running in 
        interactive or script mode.
 
-   * - :manual:`load() </reference/method/load/>`
+   * - .. _mongosh-native-method-load:
+
+       :manual:`load() </reference/method/load/>`
```

Comment:
Should this be ``load()`` ? 

I don't understand the link to the manual page.

Reply:
Thanks for noting this -- I've updated this bit.